lxd/dns/handler.go

```diff
@@ -21,23 +21,23 @@ func (d dnsHandler) ServeDNS(w dns.ResponseWriter, r *dns.Msg) {
 	if d.server.zoneRetriever == nil {
 		m := new(dns.Msg)
 		m.SetRcode(r, dns.RcodeServerFailure)
-		w.WriteMsg(m)
+		_ = w.WriteMsg(m)
```

Comment:
These errors should be logged as warnings it indicates the DNS serving functionality hasn't worked.